### ISSUE TITLE
Is this an active fork?

### ISSUE THREAD

**Issue by Davincible** [TheForgotten69/goinsta#16]
Is this fork meant to be maintained?

I analyzed the web traffic from the latest android app and noticed the login procedure is vastly different (only makes a single stand-alone call to `/login`) from the shenanigans that are going on in the login procedure here. I am looking to create a patch, but should I make a pull request here or create my own fork?

**Comment by TheForgotten69** [comment]
Hi!
The login procedure is highly dependant on the Instagram version. Currently, it's still working as it's based on the older version of Instagram API. But you can improve this aspect here.

**Comment by Davincible** [comment]
Yeah I can imagine. Current login procedure here makes a request to a seemingly depricated endpoint. The latest ig app also still calls it, but it always returns 429. As this lib quits on any error the login procedure will never work. I currently have a working login procedure, I'm working on a patch

**Comment by Davincible** [comment]
@TheForgotten69 I'm done with the patch, but I can't seem to push it to my fork of this repo. I constantly get

 ```Total 3 (delta 2), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
remote: fatal: did not receive expected object 9e325a067d92cc530eba8c268195bf66425d2a3a
```
And google also didn't really help. Never seen this before on any other repo. Do you know what's up with this?

**Comment by Davincible** [comment]
@TheForgotten69 I wasn't able to get rid of the error, so I created a fork from the main repo and uploaded my code there. https://github.com/Davincible/goinsta

Login and basic endpoints work, but not all have been migrated yet
